From e606be4bd46a741f9d8e3f31d089d66ea4a4bf40 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Wed, 15 Jul 2020 09:38:58 +0200
Subject: [PATCH 769/830] ANDROID: GKI: add CONFIG_BT to x86 gki_defconfig
X-NVConfidentiality: public

It was enabled for arm64, but for some reason disabled for x86, which
led to lots of confusion when trying to do some build testing...

Bug: 157965270
Bug: 162909109
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: Ic8e295c4d1f1cb368d386f2ba5ee1a5edbb83a96
Signed-off-by: Connor O'Brien <connoro@google.com>
---
 arch/x86/configs/gki_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 12fd835113b8..bd21f40fd889 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -184,6 +184,7 @@ CONFIG_NET_EMATCH_U32=y
 CONFIG_NET_CLS_ACT=y
 CONFIG_VSOCKETS=y
 CONFIG_BPF_JIT=y
+CONFIG_BT=y
 CONFIG_CFG80211=y
 # CONFIG_CFG80211_DEFAULT_PS is not set
 # CONFIG_CFG80211_CRDA_SUPPORT is not set
-- 
2.17.1

